modules/ROOT/pages/api-analytics-dashboard.adoc

```diff
@@ -10,14 +17,7 @@ For Gold and Platinum subscriptions to Anypoint Monitoring, API analytics data a
 . Sign into Anypoint Platform and navigate to *Anypoint Monitoring*.
 . Select the Environment, Resource name, and Version/Instance for the API to view.
 . Click *View*. +
-The API analytics page is displayed, where you can view information about a running, managed API, including: +
-* Requests
-* Requests by HTTP response code
-* Requests by method
-* Requests (Avg), Average Size of Requests, and Average Size of Responses
-* Requests by HTTP policy violation
-* Requests by failure
-+
+The API analytics page is displayed, where you can view information about a running, managed API. +
 image::monitoring-api-analytics-top.png[]
```

Comment:
For all screenshots, add a caption (line above starting with a dot and no space). Alt text goes inside the [].

Reply:
```suggestion
```

Reply:
Removing these since there are two views, and they do not help the reader.